data/test_domains/test_domain_from_directory_for_entities/game_1.yml

```diff
@@ -0,0 +1,13 @@
+version: "2.0"
+
+intents:
+  - play:
+      use_entities:
+        - ball
+        - chess
+  - stow_away:
+      use_entities: true
+
+entities:
+  - ball
+  - chess
```

Comment:
Very minor comment: I have previously been asked to add a new line at the end of yaml files in our data directory. Probably worth adding them to be consistent with existing files.